src/app/parcels/ParcelsPage.tsx

```diff
@@ -127,7 +128,19 @@ const ParcelsPage: React.FC<{}> = () => {
     const [tableData, setTableData] = useState<ParcelsTableRow[]>([]);
     const [selectedParcelId, setSelectedParcelId] = useState<string | null>(null);
     const [selected, setSelected] = useState<number[]>([]);
+    const [modalIsOpen, setModalIsOpen] = useState<boolean>(false);
     const theme = useTheme();
+    const router = useRouter();
+
+    const searchParams = useSearchParams();
+    const parcelId = searchParams.get("parcelId");
```

Comment:
Should we pull the "parcelId" out to a constant? We use it here and also in `router.push(`parcels/?parcelId=${row.data.parcelId}`);`